File: WanaD.AppServiceContract/VisitPlotCollectionData.cs
using Fs.Erp.Core;
using System;
using System.Collections.Generic;

namespace WanaD.AppServiceContract
{
    public class VisitPlotCollectionData
    {
        public Guid? Id { get; set; }
        public FarmAndPlotData Farm { get; set; }
        public NameAndIdData Plot { get; set; }
        public decimal? Score { get; set; }
        public DateTime? VisitedDate { get; set; }
        public string TestResult { get; set; }
        public decimal? BiomassPerRai { get; set; }
        public string BiomassDetails { get; set; }
        public string MonsoonStaff { get; set; }
        public string Remark { get; set; }
        public bool IsSummarized { get; set; }
        public List<FileInfoData> FileInfoList { get; set; }
    }
}

File: WanaD.WebClient/app/src/Modules/WanaD/Scripts/AppServiceContract/VisitPlotCollectionData.ts

 

 

 

    import FarmAndPlotData from 'Modules/WanaD/Scripts/AppServiceContract/FarmAndPlotData';

    import NameAndIdData from 'Modules/WanaD/Scripts/AppServiceContract/NameAndIdData';

 
class VisitPlotCollectionData 
{
    
        Id: string;
    
        Farm: FarmAndPlotData;
    
        Plot: NameAndIdData;
    
        Score: number;
    
        VisitedDate: Date;
    
        TestResult: string;
    
        BiomassPerRai: number;
    
        BiomassDetails: string;
    
        MonsoonStaff: string;
    
        Remark: string;
    
        IsSummarized: boolean;
    
        FileInfoList: any;
    
}
 
export default VisitPlotCollectionData;
 




